chia/pools/pool_wallet.py

```diff
@@ -889,6 +889,7 @@ async def claim_pool_rewards(
             )
             assert fee_tx.spend_bundle is not None
             full_spend = SpendBundle.aggregate([fee_tx.spend_bundle, claim_spend])
+            fee_tx = dataclasses.replace(fee_tx, spend_bundle=None)
```

Comment:
I don't understand why you're changing `fee_tx` here

Reply:
Yeah it was a bit of a decision I made because of a sort of bugged (but working way) it was operating before:
The way it used to work:
* fee tx would have its own spend bundle
* travel tx would get all spends attached to it including the fee spend
* travel tx would be pushed first
* the fee tx is optionally pushed

This works because travel tx gets to the mempool first and then the fee tx is just a mempool duplicate that gets rejected as such.  However, when we start pushing both transactions with the new method, we are sure to aggregate the spends from all transactions together onto a single record to make sure they are pushed at once.  This means that we had two copies of the fee spend in the spendbundle after aggregation because we had included it on both txs.

Instead of choosing not to include the fee spend on the travel tx, I just adopted the existing paradigm that most wallets use and do the (now unnecessary) aggregation and fee tx clearance before we send the transactions to the DB to get pushed.

Reply:
it might warrant a comment in the code. but I'll leave that up to you. feel free to resolve this thread